ingestion/src/metadata/utils/column_type_parser.py

```diff
@@ -146,7 +146,7 @@ def get_column_type(column_type: Any) -> str:
         if isinstance(column_type, types.NullType):
             return "NULL"
         for sql_type in ColumnTypeParser._SOURCE_TYPE_TO_OM_TYPE.keys():
-            if str(column_type) in sql_type:
+            if str(column_type) == sql_type:
```

Comment:
@ayush-shah is this the same problem in hive connector as well

Reply:
@harshach no, this is not the same as hive, as we are checking it with the _SOURCE_TYPE_TO_OM_TYPE whereas in the hive, we are testing hive out with _COLUMN_TYPE_MAPPING.